src/core/lib/security/credentials/composite/composite_credentials.h

```diff
@@ -97,6 +64,9 @@ class grpc_composite_channel_credentials : public grpc_channel_credentials {
 
 class grpc_composite_call_credentials : public grpc_call_credentials {
  public:
+  typedef grpc_core::InlinedVector<
```

Comment:
[nit and optional] I would use `using`  instead of `typedef`.

Reply:
Done.